src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,34 @@
+.card {
+  &__stars {
+    display: flex;
+    align-items: flex-end;
```

Comment:
The use of 'flex-end' in align-items is not suitable in this context. The stars should be aligned in the center vertically. You should use 'center' instead of 'flex-end'.